### PR TITLE
Create breed-bird-survey-nlcd.json

### PR DESCRIPTION
**Pull request by shreyneil**
The following pull request is regarding the following issue https://github.com/weecology/retriever/issues/117.
Please verify the changes and get back.

### CODE REVIEW

**Comment by henrykironde** [comment]
@ethanwhite could you have a look at this PR based on the Request 117.
I do think we have to add the files(if not already added) in the current breading bird survey dataset, which was bbs previously.

**Comment by henrykironde** [comment]
@shreyneil, I will get back to you regarding this dataset after discussing with the Team

**Comment by ethanwhite** [comment]
Sorry for the delay in getting to this. I think that we should treat this as a separate dataset from `breed-bird-survey` with a related name like `breed-bird-survey-nlcd` or `breed-bird-survey-landcover`. Then users can join them together or we can automated the joining in the weaver.

I think we should use the figshare versions of these files instead of the old Ecological Archives ones. @shreyneil - for back story on this Ecological Archives moved all of their content to figshare. The old site still runs, but sometimes goes down for weeks at a time. The same files are available here:

https://figshare.com/articles/Data_Paper_Data_Paper/3554424

The description should be something like:

> Landcover data for all North American Breeding Bird Survey routes from the 2006 National Land Cover Database at buffers from 200 m to 10 km.

**Comment by zhangcandrew** [comment]
Hey @shreyneil!

I just went through this and downloaded the script locally. In PR #1042, we completely re-did how we handle scripts, so the way this script is setup no longer works. I suspect all you need to do though is fill in the resources in the json with the appropriate values from the csv files. You can read more about creating scripts and filling in resource values at our [docs found here](http://retriever.readthedocs.io/en/latest/scripts.html?highlight=scripts). 

Otherwise, to test with the updated version of how we handle scripts, you'll need to bring this branch up to date with the retriever repository master branch. 

Let me know if you have any questions!

**Comment by shreyneil** [comment]
@zhangcandrew  Done with the changes, Please verify.

**Comment by zhangcandrew** [comment]
@shreyneil, looks like they might have changed the name for the file in NLCD 400 meter buffer - all the data is showing up blank when downloaded through sqlite.

**Comment by ss-is-master-chief** [comment]
The JSON file version is 1.0.0. Will it be meeting the minimum retriever requirements?

**Comment by shreyneil** [comment]
@zhangcandrew  sir i verified,the name of the file is same as on the figshare site https://figshare.com/articles/Data_Paper_Data_Paper/3554424 . There must be some other problem. Can you please review again, that would be great.

**Comment by zhangcandrew** [comment]
@shreyneil, the items in Configuration_metrics_400_m_buffer.txt are tab separated, not comma separated, so none of the data was being downloaded or processed.

**Comment by shreyneil** [comment]
Changed the dialect from , to tabs ,Please check again.

**Comment by henrykironde** [comment]
@shreyneil could you give me the results you get if you `retriever install sqlite breed-bird-survey-nlcd` looks like it fails here

File: scripts/breed-bird-survey-nlcd.json [hunk 1]
```diff
@@ -788,7 +788,7 @@
     {
       "path": "BBS_NLCD_400_m_buffer.txt",
       "dialect": {
-        "delimiter": ",",
+        "delimiter": "  ",
```
Comment (henrykironde) [inline scripts/breed-bird-survey-nlcd.json]: wrong file

Comment (henrykironde) [reply scripts/breed-bird-survey-nlcd.json]: Configuration_metrics_400_m_buffer.txt  vs BBS_NLCD_400_m_buffer.txt

File: scripts/breed-bird-survey-nlcd.json [hunk 2]
```diff
@@ -0,0 +1,899 @@
...
+          }
+        ]
+      },
+      "url": "https://ndownloader.figshare.com/articles/3554424/versions/1"
+    }
+  ],
+  "retriever": "True",
+  "retriever_minimum_version": "2.0.dev",
+  "title": "Breed-Bird-Survey-nlcd Data ",
+  "urls": {},
```
Comment (henrykironde) [inline scripts/breed-bird-survey-nlcd.json]: remove this line . "urls": {},

**Comment by henrykironde** [comment]
squash all commit and rebase to master

**Comment by shreyneil** [comment]
Done sir , Please verify.

**Comment by henrykironde** [comment]
One last think. For this dataset remove the delimiter. Retriever will find automatically detect it

```
      "path": "Configuration_metrics_400_m_buffer.txt",
      "dialect": {
        "header_rows": 1
      },
      "name": "Configuration_metrics_400_m_buffer",
      "schema": {
        "fields": [

```
And squash the commits to one. 
update your master first. 
```
git checkout master
git fetch upstream
git merge upstream/master
```


then 

```
git checkout patch-8
git reset --soft bbd721a
git commit --amend 
git rebase master 
```
run
retriever reset scripts
git clean -fx 
python version.py  to make sure the version.txt is updated
if the version.txt  has changed, add and do a commit --amend

**Comment by shreyneil** [comment]
Done with the changes. Please verify.

**Comment by henrykironde** [comment]
@shreyneil remove only the delimiter for this file Configuration_metrics_400_m_buffer.txt.
or any tab files. You just discovered a bug and I have created and issue for that :+1:

**Comment by shreyneil** [comment]
Done sir !!!

**Comment by henrykironde** [comment]
I have tested this PR and I think it is ready. @zhangcandrew

**Comment by zhangcandrew** [comment]
Tested it out, looks good to me!

If anything arises, feel free to re-open.

**Pull request merged**